test/compilable/issue20915.d

```diff
@@ -0,0 +1,10 @@
+/*
+REQUIRED_ARGS: -version=tralala
+*/
+module issue20915;
+
+version = custom;
+static assert (is(custom  == version));
+static assert (is(tralala == version));
+static assert (is(linux   == version) || is(posix == version));
```

Comment:
You will obviously need to ensure this test is not run on windows.

Reply:
yes, fixed now.